scorum/chain.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "scorum",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Scorum Network",
+  "chain_id": "scorum-1",
+  "bech32_prefix": "scorum",
+  "daemon_name": "scorumd",
+  "node_home": "$HOME/.scorum",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "gas",
+        "fixed_min_gas_price": 1,
+        "low_gas_price": 1,
+        "average_gas_price": 1,
+        "high_gas_price": 1
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "nsp"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/scorum/cosmos-network",
+    "recommended_version": "v1.0.0",
+    "compatible_versions": [
+      "v1.0.0"
+    ],
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/scorum/cosmos-network/main/genesis/mainnet/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ]
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
```

Comment:
This images do not exist. Please update the reference (same with the `images` property

Reply:
Uploaded logo for the chain